applications/etf_network.md

```diff
@@ -0,0 +1,292 @@
+With an eye on the future and a strong curiosity to comprehend the technologies behind cryptocurrencies, such as Blockchain, and explore their potential applications in another fields, I've taken steps into the world of web3 and blockchain by attending  the Polkadot Blockchain Academy in Berkeley, California, where I get the essential skills to navigate this revolutionary domain.
+
+### Team Code Repos
+
+The intended repos for this project are (but possibly subject to name change): 
+
+https://github.com/ideal-lab5/etf-node
+https://github.com/ideal-lab5/etf-consensus 
+https://github.com/ideal-lab5/etf-sdk
+https://github.com/ideal-lab5/etf.js
+https://github.com/ideal-lab5/etf-auction-ui
+https://github.com/ideal-lab5/contracts
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/driemworks
+- https://github.com/carloskiron
+- https://github.com/Valentinaga1
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/tony-riemer/
+- https://www.linkedin.com/in/cmonvel/
+- https://www.linkedin.com/in/valentinaga1/
+
+## Development Status :open_book:
+
+- This proposal is a continuation of our first ETF Network [grant](https://github.com/w3f/Grants-Program/blob/master/applications/cryptex.md).
+- The ETF Network documentation, and its current development status can be found [here](https://ideal-lab5.github.io/).
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 16 weeks
+- **Full-Time Equivalent (FTE):**  3
+- **Total Costs:** 90,000
+
+
+The following items are mandatory for each milestone:
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| **0e.** | Article | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). (Content, language and medium should reflect your target audience described above.) |
+
+It should also be acknowledged that we will strive for > 80% code coverage on all new lines of code. It should be assumed that each deliverable in each milestone, unless otherwise stated, include development, testing, and documentation of the deliverable.

```

Comment:
Thanks for the application @driemworks could you please adhere to the [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) and integrate the default deliverables into the actual milestones? We ask because the tests might change depending on the milestone. Ideally you would take out the text and replace with your own. 

- For example, M1 might have Rust unit tests for the substrate modules and TEE, whereas M3 might have testnet e2e tests, and M4 might have Cypress or Jest tests.
- What kind of test suites/frameworks will you be using?
- Is Docker really needed for all four milestones?
- Can you include more technical details in the milestones, such as what languages the different components will be written in.

Reply:
@keeganquigley thanks a lot for the feedback. We will take care of this to fully adhere to the application template.

Reply:
Hey @keeganquigley, I updated the document to make the default deliverables specific for each milestone. I also updated the milestones to elaborate on languages, frameworks, or other tech that we intend to use and along with more information on our approaches to testing.